diff --git a/lib/module/unstable/NativeBottomTabView.native.js b/lib/module/unstable/NativeBottomTabView.native.js
index 20a6a5cf5694d2633ce26f8960d8d53cebbfc082..1d59a2bd3df0ba3de323b3dc83b4c0441904d13d 100644
--- a/lib/module/unstable/NativeBottomTabView.native.js
+++ b/lib/module/unstable/NativeBottomTabView.native.js
@@ -263,6 +263,25 @@ export function NativeBottomTabView({
           tabBarItemTitleFontColor: activeTintColor,
           tabBarItemIconColor: activeTintColor
         };
+        const selectedTabItemAppearanceIOS = {
+          ...tabItemAppearance,
+          tabBarItemTitleFontColor: activeTintColor,
+          tabBarItemIconColor: activeTintColor
+        };
+        const tabBarItemLayoutAppearance = {
+          stacked: {
+            normal: tabItemAppearance,
+            selected: selectedTabItemAppearanceIOS
+          },
+          inline: {
+            normal: tabItemAppearance,
+            selected: selectedTabItemAppearanceIOS
+          },
+          compactInline: {
+            normal: tabItemAppearance,
+            selected: selectedTabItemAppearanceIOS
+          }
+        };
         const icon = typeof tabBarIcon === 'function' ? getPlatformIcon(tabBarIcon({
           focused: false
         })) : tabBarIcon != null ? getPlatformIcon(tabBarIcon) : undefined;
@@ -324,29 +343,13 @@ export function NativeBottomTabView({
               tabBarBackgroundColor,
               tabBarShadowColor,
               tabBarBlurEffect,
-              stacked: {
-                normal: tabItemAppearance
-              },
-              inline: {
-                normal: tabItemAppearance
-              },
-              compactInline: {
-                normal: tabItemAppearance
-              }
+              ...tabBarItemLayoutAppearance
             },
             standardAppearance: {
               tabBarBackgroundColor,
               tabBarShadowColor,
               tabBarBlurEffect,
-              stacked: {
-                normal: tabItemAppearance
-              },
-              inline: {
-                normal: tabItemAppearance
-              },
-              compactInline: {
-                normal: tabItemAppearance
-              }
+              ...tabBarItemLayoutAppearance
             },
             overrideScrollViewContentInsetAdjustmentBehavior
           },
