diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -312,1 +312,2 @@
-    sed '/^extension __ObjC\./,/^}/d;/^@usableFromInline$/{N;/_ConstraintThatIsNotPartOfTheAPIOfThisLibrary/d;};/_ConstraintThatIsNotPartOfTheAPIOfThisLibrary/d' "$swiftinterface" > "$stripped_swiftinterface"
+    # Newer Swift compilers use :: instead of . for C++ namespace separators.
+    sed '/^extension __ObjC[.:]/,/^}/d;/^@usableFromInline$/{N;/_ConstraintThatIsNotPartOfTheAPIOfThisLibrary/d;};/_ConstraintThatIsNotPartOfTheAPIOfThisLibrary/d' "$swiftinterface" > "$stripped_swiftinterface"
